bundles/binding/org.openhab.binding.zwave/database/evolve/ltm-5.xml

```diff
@@ -0,0 +1,157 @@
+			<Default>1</Default>
+			<Size>1</Size>
+			<Label lang="en">Ignore Start Level (Transmitting)</Label>
+			<Help lang="en">Dimmers controlled by this switch will start dimming from their current level.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">No</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Yes</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>3</Index>
+			<Type>list</Type>
+			<Default>1</Default>
+			<Size>1</Size>
+			<Label lang="en">Night Light</Label>
+			<Help lang="en">In night-light mode the LED on the switch will turn ON when the switch is turned OFF.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">No</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Yes</Label>
+			</Item>
+		</Parameter>
+			<Parameter>
+			<Index>4</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Invert Switch</Label>
+			<Help lang="en">Change the top of the switch to OFF and the bottom of the switch to ON.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">No</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Yes</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>5</Index>
+			<Type>list</Type>
+			<Default>1</Default>
+			<Size>1</Size>
+			<Label lang="en">Ignore Start-Level (Receiving)</Label>
+			<Help lang="en">The dimmer will start dimming from its current level.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">No</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Yes</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>19</Index>
+			<Type>list</Type>
+			<Default>2</Default>
+			<Size>1</Size>
+			<Label lang="en">Transmission Indication</Label>
+			<Help lang="en">Flicker LED while transmitting.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">No Flicker</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Flicker Whole Transmission</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">Flicker 1 Second</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>20</Index>
+			<Type>byte</Type>
+			<Default>2</Default>
+			<Minimum>1</Minimum>
+			<Maximum>255</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Polling Interval</Label>
+			<Help lang="en">How often, in minutes, to poll to keep synchronized with group.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>21</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Poll First Node</Label>
+			<Help lang="en">Poll only the first node in Group 1.</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">No</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Yes</Label>
+			</Item>			
+		</Parameter>
+	</Configuration>
+
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>5</Maximum>
+			<Label lang="en">Group 1</Label>
```

Comment:
I think I could see how this might be useful. For my current setup, OpenHAB won't be handling any signals from the switch; the switch is just associated with my dimmer and it controls the dimmer directly independently of the controller. If I understand correctly, this tag would make it so that the controller is automatically associated into the group so that the switch will send any switching commands to the controller?